scripts/run-sonar-scanner-cli.sh

```diff
@@ -21,16 +21,41 @@ if [[ -n "${INPUT_PROJECTBASEDIR}" ]]; then
   scanner_args+=("-Dsonar.projectBaseDir=${INPUT_PROJECTBASEDIR}")
 fi
 
+# The SSL folder may exist on an uncleaned self-hosted runner
+SONAR_SSL_FOLDER=~/.sonar/ssl
+# Use keytool for now, as SonarQube 10.6 and below doesn't support openssl generated keystores
+# keytool require a password > 6 characters, so we wan't use the default password 'sonar'
+KEYTOOL_MAIN_CLASS=sun.security.tools.keytool.Main
+SONAR_SSL_TRUSTSTORE_FILE="$SONAR_SSL_FOLDER/truststore.p12"
+SONAR_SSL_TRUSTSTORE_PASSWORD=changeit
+
+if [ -f "$SONAR_SSL_TRUSTSTORE_FILE" ]; then
+  echo "::warning title=SonarScanner: Removing 'sonar' alias from already existing Scanner truststore: $SONAR_SSL_TRUSTSTORE_FILE"
+  "$SONAR_SCANNER_JRE/bin/java" "$KEYTOOL_MAIN_CLASS" \
```

Comment:
I guess you have tested, but what happens if the file actually exists but there is no 'sonar' alias? Is it silently continuing?

Reply:
I am still in the process of making changes - I am testing with CI because it's not super-easy for me to test locally, given WSL instability.
I am adding a more extensive QA test with this scenario in it.

Reply:
Ok, I think a fixed the behavior. 

Now, if the `truststore.p12` exists already, we don't remove anymore the `~/sonar/ssl` directory. Instead, we check whether such `truststore.p12` contains a `sonar` alias. If it does, and only if it does, we remove the alias from the truststore, and then we proceed as usual.

Here is a successful run: https://github.com/SonarSource/sonarqube-scan-action/actions/runs/12355702818/job/34479954427?pr=165

Could you please check whether the behavior makes sense for you?